src/test/java/org/springframework/data/couchbase/domain/Address.java

```diff
@@ -0,0 +1,43 @@
+package org.springframework.data.couchbase.domain;
+
+import org.springframework.data.annotation.Id;
+import org.springframework.data.couchbase.config.AbstractCouchbaseConfiguration;
+import org.springframework.data.couchbase.core.mapping.Document;
+import org.springframework.data.couchbase.core.mapping.id.GeneratedValue;
+import org.springframework.data.couchbase.core.mapping.id.GenerationStrategy;
+
+import java.util.UUID;
+
+@Document
+public class Address extends AbstractEntity {
```

Comment:
I see the entity changes, but can you add a automated test case for this so we catch any regressions?